lua/lspconfig/pasls.lua

```diff
@@ -0,0 +1,23 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.pasls = {
+  default_config = {
+    cmd = { 'pasls' },
+    cmd_env = {
+      PP = '/usr/lib/fpc/3.2.2/ppcx64',
```

Comment:
This should be documented at the least, because I doubt this paths are portable (especially not to mac and windows)

Reply:
I added Documentation to everyone of those